lib/status/timeline.go

```diff
@@ -39,3 +43,43 @@ func Timeline(ctx context.Context) (*pb.TimelineResponse, error) {
 	}()
 	return client.Timeline(ctx, &pb.TimelineRequest{})
 }
+
+// PrintEvent prints the event to the provided writer.
+func PrintEvent(w io.Writer, event *pb.TimelineEvent) {
+	timestamp := event.GetTimestamp().ToTime().Format(time.RFC3339)
+	switch event.GetData().(type) {
+	case *pb.TimelineEvent_ClusterDegraded:
+		fmt.Fprintln(w, color.RedString("%s [Cluster Degraded]",
+			timestamp))
+	case *pb.TimelineEvent_ClusterHealthy:
+		fmt.Fprintln(w, color.GreenString("%s [Cluster Healthy]",
+			timestamp))
+	case *pb.TimelineEvent_NodeAdded:
+		fmt.Fprintln(w, color.YellowString("%s [Node Added]\tnode=%s",
+			timestamp, event.GetNodeAdded().GetNode()))
+	case *pb.TimelineEvent_NodeRemoved:
+		fmt.Fprintln(w, color.YellowString("%s [Node Removed]\tnode=%s",
+			timestamp, event.GetNodeRemoved().GetNode()))
+	case *pb.TimelineEvent_NodeDegraded:
+		fmt.Fprintln(w, color.RedString("%s [Node Degraded]\tnode=%s",
+			timestamp, event.GetNodeDegraded().GetNode()))
+	case *pb.TimelineEvent_NodeHealthy:
+		fmt.Fprintln(w, color.GreenString("%s [Node Healthy]\tnode=%s",
+			timestamp, event.GetNodeHealthy().GetNode()))
+	case *pb.TimelineEvent_ProbeFailed:
+		e := event.GetProbeFailed()
+		fmt.Fprintln(w, color.RedString("%s [Probe Failed]\tnode=%s\tchecker=%s",
+			timestamp, e.GetNode(), e.GetProbe()))
+	case *pb.TimelineEvent_ProbeSucceeded:
+		e := event.GetProbeSucceeded()
+		fmt.Fprintln(w, color.GreenString("%s [Probe Succeeded]\tnode=%s\tchecker=%s",
+			timestamp, e.GetNode(), e.GetProbe()))
+	case *pb.TimelineEvent_LeaderElected:
+		e := event.GetLeaderElected()
+		fmt.Fprintln(w, color.YellowString("%s [Leader Elected]\tprev=%s\tnew=%s",
```

Comment:
Won't an empty `prev` be puzzling? This is supposed to be human-readable - we can shorten and abbreviate the fields in the machine output
```
2020-05-06T12:25:12Z [Leader Elected]   prev=                   new=172.28.128.101
```
vs
```
2020-05-06T12:25:12Z [Leader Elected]   new leader 172.28.128.101
2020-05-06T12:25:15Z [Leader Elected]   leader changed 172.28.128.101 -> 172.28.128.102
```
Another nit is whether we should be showing bogus changes like 172.28.128.101 -> 172.28.128.101 or is it just for display purposes in the PR?

Reply:
Updated in cfb45e5
Instead of filtering same leader elected events in gravity I've created PR in planet to ignore these events. https://github.com/gravitational/planet/pull/635